LawnCare/android/app/src/main/java/com/redflag/lawncare/calculation/CalculationFragment.java

```diff
@@ -0,0 +1,59 @@
+package com.redflag.lawncare.calculation;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.google.android.material.textfield.TextInputLayout;
+import com.redflag.lawncare.R;
+
+public class CalculationFragment extends Fragment {
+
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_calculation,
+                container, false);
+
+
+        Button submit = view.findViewById(R.id.SubmitButton);
+        submit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                TextInputLayout input1_TIL = view.findViewById(R.id.input_1);
```

Comment:
Might be nit picking here, but i do think we should add more descriptive names to R.id.input_1. Like r.id.area or obstruction for example, it'll save us a potential headache in the future I think.

Reply:
fixed!